src/DocumentationBrowserViewExtension/DocumentationBrowserUtils.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.IO;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace Dynamo.DocumentationBrowser
+{
+    /// <summary>
+    /// Common utilities used across classes in the DocumentationBrowserViewExtension.
+    /// </summary>
+    internal static class DocumentationBrowserUtils
+    {
+        private const string SCRIPT_TAG_REGEX = @"<script[^>]*>[\s\S]*?</script>";
+        private const string DPISCRIPT = @"<script> function getDPIScale()
+        {
+            var dpi = 96.0;
+            if (window.screen.deviceXDPI != undefined)
+            {
+                dpi = window.screen.deviceXDPI;
+            }
+            else
+            {
+                var tmpNode = document.createElement('DIV');
+                tmpNode.style.cssText = 'width:1in;height:1in;position:absolute;left:0px;top:0px;z-index:99;visibility:hidden';
+                document.body.appendChild(tmpNode);
+                dpi = parseInt(tmpNode.offsetWidth);
+                tmpNode.parentNode.removeChild(tmpNode);
+            }
+
+            return dpi / 96.0;
+        }
+
+        function adaptDPI()
+        {
+            var dpiScale = getDPIScale();
+            document.body.style.zoom = dpiScale;
+
+            var widthPercentage = ((100.0 / dpiScale)-5).toString() + '%';
+            document.body.style.width = widthPercentage;
+        }
+        adaptDPI() </script>";
+
+        /// <summary>
+        /// Clean up possible script tags from the content string.
+        /// </summary>
+        /// <param name="content"></param>
+        /// <returns>Returns true if any tags was removed from the content</returns>
+        internal static bool RemoveScriptTagsFromString(ref string content)
+        {
+            if (Regex.IsMatch(content, SCRIPT_TAG_REGEX, RegexOptions.IgnoreCase))
```

Comment:
Script tags aren't the only risk we need to sanitize, and we don't want to be doing our own HTML parsing -- there are a lot of edge cases. Consider a library like https://github.com/mganss/HtmlSanitizer instead.

Reply:
More: http://htmlparsing.com/regexes.html

Reply:
changed this to use `HtmlSanitizer` it seems like this library has a lot of configuration options, my knowledge on what might be considered dangerous in this is limited, so for know im just using the default settings.

Reply:
this brings in a bunch of dependencies - @QilongTang @mmisol any thoughts?

Reply:
hmm actually I would rather not bring in more dependencies, is this a must have for locally hosted static web content? @Dewb

Reply:
@SHKnudsen - could you look into potential html sanitization options that have fewer dependencies?

Reply:
@mjkkirschner i looked into this and it seems like there arent a lot of good options, here is what i found so far:

[**AntiXss**](https://docs.microsoft.com/en-us/dotnet/api/system.web.security.antixss.antixssencoder?view=netframework-4.8) - looks like this hasent been update since 2014, also read a few negative articles about it. (https://www.nuget.org/packages/AntiXSS/)

[**HtmlRuleSanitizer**](https://github.com/Vereyon/HtmlRuleSanitizer) - This seems to be the most used option after *HtmlSanitizer*, however this still has dependecies on an Html Parser, in this case that is [HtmlAgilityPack](https://www.nuget.org/packages/HtmlAgilityPack/). So this would probably bring in about the same dependencies as **HtmlSanitizer** 

[**HtmlSanitizer**](https://github.com/mganss/HtmlSanitizer) - A @Dewb suggested, this looks to be the most commonly used library. It depends on [AngelSharp](https://github.com/AngleSharp/AngleSharp) which also looks like its the most commonly use Html parser.

In addition to these three the only real option i found was to [build our own sanitizer](https://weblog.west-wind.com/posts/2012/Jul/19/NET-HTML-Sanitation-for-rich-HTML-Input), however this would probably still bring in a dependency on a HtmlParser.

I dont have a lot of experience on this area tbh, so not entirely sure what we actually need the sanitation to do except for disabling `<script>` tags.

Reply:
thanks for looking into this @SHKnudsen - it looks like in the latest releases of HtmlSanitizer the `system.encoding.codepages dependency` was removed - perhaps we could use that newest release?

Reply:
@mjkkirschner looks like its still there:

**HtmlSanitizer Dependencies**
<img width="365" alt="SanitizerDep" src="https://user-images.githubusercontent.com/13732445/95484494-72228800-0988-11eb-8fd5-677672293450.png">

**Markdig Dependencies**
<img width="365" alt="MarkdigDep" src="https://user-images.githubusercontent.com/13732445/95484544-81093a80-0988-11eb-8a33-1faf2127dc73.png">